data/json/monsters.json

```diff
@@ -2367,6 +2394,7 @@
     "armor_bash":2,
     "armor_cut":2,
     "vision_day":50,
+    "vision_night":5,
```

Comment:
Necro and master having boosted vision makes sense, since they're optimized for support and therefore it'd help for them to see what's going on.